BUILD.bazel

```diff
@@ -16,6 +16,29 @@ package(default_visibility = [
     "//visibility:private",
 ])
 
+proto_library(
+    name = "stats_proto",
+    srcs = ["grpc/core/stats.proto"],
+)
+
+proto_library(
+    name = "benchmark_service_proto",
+    srcs = [
+        "grpc/testing/benchmark_service.proto",
+        "grpc/testing/messages.proto",
```

Comment:
Messages.proto should not be in srcs here. It should get its own proto_library, since it will be used by test.proto as well.

Reply:
Done